build.gradle

```diff
@@ -18,19 +17,43 @@ task buildDocumentation(type: Exec) {
     args '-c', "docker run --volume $rootDir/docs:/docs --rm squidfunk/mkdocs-material:5.2.2 build"
 }
 
+static getCustomArtifactId(project) {
+    if (project.name == 'gradle-plugin') {
```

Comment:
Can't we rename the project instead?

Reply:
No because there is already a `gradle` directory.

Reply:
You would OK to have the Gradle wrapper and the Gradle plugin code in the same dir?

Reply:
Good point :D